packages/combobox/src/LionCombobox.js

```diff
@@ -284,6 +340,16 @@ export class LionCombobox extends OverlayMixin(LionListbox) {
      * @private
      */
     this.__prevCboxValue = '';
+    /**
```

Comment:
where do these lines come from ?

Reply:
I think you should manually override the _onFilterMatch and _onFilterUnmatch methods (and add the methods and vars on top)